server/pnpm-lock.yaml

```diff
@@ -5,9 +5,9 @@ settings:
   excludeLinksFromLockfile: false
 
 dependencies:
-  body-parser:
```

Comment:
is there any reason for removing body-parser?

Reply:
no need, I removed it from package.json and forgot that pnpm.lock file will also e exported.

Reply:
got it so we don't need to install another dependency of body-parser for Express 4.16+ if I'm getting it right.